### PR TITLE
add Request for quote object and update add_ons to have a price

### PR DESCRIPTION
**Pull request by jeeppeck**
update AddOns object for quotes to be able to handle p3l driven prices. Also added a request for quote object while I was there.

### CODE REVIEW

File: paperless/client.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class PaperlessClient(object):
 
     __instance = None
     access_token = None
-    base_url = "https://api.paperlessparts.com"
+    base_url = "http://localhost/api"
```
Comment (rcarrington) [inline paperless/client.py:20]: @jeeppeck You should change this back

Comment (jeeppeck) [reply paperless/client.py:20]: woah could have sworn I changed that before pushed, thanks for pointing that out

**Comment by rcarrington** [comment]
This looks good to me, once you resolve the merge conflict.

**Comment by jeeppeck** [comment]
> This looks good to me, once you resolve the merge conflict.

Should be good now

**Review by 10flow — APPROVED** [review]

**Pull request merged**